src/Appwrite/Platform/Tasks/Restore.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Exception;
+use Utopia\App;
+use Utopia\DSN\DSN;
+use Utopia\Platform\Action;
+use Utopia\CLI\Console;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\DOSpaces;
+use Utopia\Storage\Device\Local;
+use Utopia\Validator\Text;
+use Utopia\Validator\WhiteList;
+
+class Restore extends Action
+{
+    protected ?DSN $dsn = null;
+    protected string $database;
+    public const PROCESSORS = 4;
+
+    public static function getName(): string
+    {
+        return 'restore';
+    }
```

Comment:
Let's move all functions after the constructor

Reply:
Done!